packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -20,12 +20,14 @@ function assertServerError (res) {
 export default class AwsS3Multipart extends BasePlugin {
   static VERSION = packageJson.version
 
+  #client
+
   constructor (uppy, opts) {
     super(uppy, opts)
     this.type = 'uploader'
     this.id = this.opts.id || 'AwsS3Multipart'
     this.title = 'AWS S3 Multipart'
-    this.client = new RequestClient(uppy, opts)
+    this.#client = new RequestClient(uppy, opts)
```

Comment:
While this change is desirable, it's also a breaking change (in the case someone out there is relying on `client` being public. Could you add a getter and a setter, with a TODO comment asking to remove those on the next major?

Reply:
Good point. I've added them. Thanks

Reply:
Fixed. Good catch :)